core/new-gui/src/environments/version.ts

```diff
@@ -1,14 +0,0 @@
-// IMPORTANT: THIS FILE IS AUTO GENERATED! DO NOT MANUALLY EDIT OR CHECKIN!
```

Comment:
is this mis-deleted?

Reply:
No. I deleted it. It is not needed. Since every time, we will anyways be doing `npm install` before launching gui

Reply:
@avinash0161 maybe unrelated to this PR, but why does the version number always show `dirty`?